### DISCUSSION TITLE
The total size of all Environment Variables (4.03KB) exceeds 4KB.

### DISCUSSION THREAD

**Discussion by vbh689** [spencerwooo/onedrive-vercel-index#47] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/22710598/129436117-8eff4d5d-9b88-4f59-9ec0-a4360e2e2b16.png)

I'm using tokens from rclone, any way to fix this?
Thank you.

**Comment by spencerwooo** [comment, marked as answer]
Dup of issue #19, basically no better option than to deploy on Netlify.
